### PR TITLE
Add basic example of NIM with Run.ai inference

### PR DESCRIPTION
**Pull request by mlsorensen**

### CODE REVIEW

**Comment by mlsorensen** [comment]
Sorry about the tweaks, this should be ready for review.

File: run.ai/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# NIMs on Run.ai
+
+Run.ai provides a fast and efficient platform for running AI workloads. It sits on top of a group of Kubernetes clusters and provides UI, GPU-aware scheduling, container orchestration, node pooling, organizational resource quota management, and more. It gives customers the tools to manage resources across multiple Kubernetes clusters and subdivide them across project and departments, and automates Kubernetes primitives with its own AI optimized resources.
+
```
Comment (resker) [inline run.ai/README.md:4]: <! -- Suggested re-wording & additions: --> 

[Run.ai](https://www.run.ai) provides a platform for accelerating AI development delivering life cycle support spanning from concept to deployment of AI workloads. It layers on top of Kubernetes starting with a single cluster but extending to centralized multi-cluster management.  It provides UI, GPU-aware scheduling, container orchestration, node pooling, organizational resource quota management, and more. And it offers administrators, researchers, and developers tools to manage resources across multiple Kubernetes clusters and subdivide them across project and departments, and automates Kubernetes primitives with its own AI optimized resources.

## Deployment Locations

The Run:ai Control Plane is available as a [hosted service](https://docs.run.ai/v2.18/home/components/#runai-control-plane-on-the-cloud) or alternatively as a [self-hosted](https://docs.run.ai/v2.18/home/components/#self-hosted-control-plane) option (including in disconnected "air-gapped" environments).  In either case, the control plane can manage Run:ai "cluster engine" equipped clusters whether local or remotely cloud hosted. 

<! -- Where is the DGX Cloud Run:ai control plane hosted?  This should be referenced here -->

File: run.ai/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+## InferenceWorkload
+
+Run.ai provides an [InferenceWorkload](https://docs.run.ai/latest/Researcher/workloads/inference-overview/) resource to help automate inference services like NIMs. It leverages Knative to automate the underlying service and routing of traffic.
+
```
Comment (resker) [inline run.ai/README.md:8]: [Knative](https://github.com/knative)

File: run.ai/README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# NIMs on Run.ai
```
Comment (resker) [inline run.ai/README.md:1]: Could you move the run.ai directory & contents under docs?

File: run.ai/README.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+This example can also be deployed through [UI](https://docs.run.ai/latest/Researcher/workloads/inference-overview/) - including creating the secret and InferenceWorkload.
+
+**Prerequisites**:
+* A Runai Project (and corresponding Kubernetes namespace, which is the project name prefixed with `runai-`). You should be set up to run "kubectl" commands to the target cluster and namespace.
+* An NGC API Key
+* `curl` and `jq` for the test script
+* A Docker registry secret for `nvcr.io` needs to exist in your Run.ai project. This can only be created through the UI, via "credentials" section. Add a new docker-registry credential, choose the scope to be your project, set username to `$oauthtoken` and password to your NGC API key. Set the registry url to `ngcr.io`. This only has to be done once per scope, and Run.ai will detect and use it when it is needed.
+
+1. Deploy InferenceWorkload to your current Kubernetes context via Helm, with working directory being the same as this README, setting the neccessary environment variables
```
Comment (resker) [inline run.ai/README.md:23]: typo, should be "necessary"

File: run.ai/README.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+Run.ai provides an [InferenceWorkload](https://docs.run.ai/latest/Researcher/workloads/inference-overview/) resource to help automate inference services like NIMs. It leverages Knative to automate the underlying service and routing of traffic.
+
+It should be noted that InferenceWorkload is an optional add-on for Run.ai. Consult your Run.ai UI portal or cluster administrator to determine which clusters support InferenceWorkload.
+
+### Basic Example
+
+At the core, running NIMs with InferenceWorkload is quite simple. However, many customizations are possible, such as adding variables, PVCs to cache models, health checks, and other special configurations that will pass through to the pods backing the services. The `examples` directory can evolve over time with more complex deployment examples. The following example is a bare minimum configuration.
+
+This example can also be deployed through [UI](https://docs.run.ai/latest/Researcher/workloads/inference-overview/) - including creating the secret and InferenceWorkload.
+
```
Comment (resker) [inline run.ai/README.md:16]: Suggest rewording the following "prerequisites" to "example preparation". Also note that the NGC API key portion is referenced in the new prerequisites section above.

File: run.ai/README.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+### PVC Example
+
+The PVC example runs in much the same way. It adds a mounted PVC to the example NIM container in a place where it can be used as a cache - `/opt/nim/.cache`, and configured to be retained between helm uninstall and install, so that the model data need only be downloaded on first use.
+
+```
+% helm install --set namespace=$NAMESPACE --set ngcKey=$NGC_KEY my-llama-pvc examples/basic-llama-pvc
+
+% kubectl get ksvc basic-llama-pvc --watch
+```
+
```
Comment (resker) [inline run.ai/README.md:74]: ### Air-gapped operations

For scenarios in which Run:ai clusters are operating in air-gapped (disconnected) environments, please see NVIDIA NIM documentation for [serving models from local assets](https://docs.nvidia.com/nim/large-language-models/latest/getting-started.html#serving-models-from-local-assets).

**Review by resker — CHANGES_REQUESTED** [review]
Apologies... probably should have trickled these in...

File: run.ai/README.md (hunk 1 above)
Comment (mlsorensen) [reply run.ai/README.md:4]: Can you double check those links on item 5? @resker

Comment (mlsorensen) [reply run.ai/README.md]: I think I got it figured out, it was the extra bracket and parentheses after that I was questioning.

**Review by resker — APPROVED** [review]
+1 non-binding

File: docs/run.ai/README.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+## Prerequisites
+
+1. A conformant Kubernetes cluster ([RunAI K8s version requirements](https://docs.run.ai/latest/admin/overview-administrator/))
+2. RunAI Control Plane and cluster(s) [installed](https://docs.run.ai/latest/admin/runai-setup/cluster-setup/cluster-install/) and operational
+3. [NVIDIA GPU Operator](https://github.com/NVIDIA/gpu-operator) installed
+4. General NIM requirements: [NIM Prerequisites](https://docs.nvidia.com/nim/large-language-models/latest/getting-started.html#prerequisites)
+5. An NVIDIA AI Enterprise (NVAIE) License: [Sign up for NVAIE license](https://build.nvidia.com/meta/llama-3-8b-instruct?snippet_tab=Docker&signin=true&integrate_nim=true&self_hosted_api=true) or [Request a Free 90-Day NVAIE License](https://enterpriseproductregistration.nvidia.com/?LicType=EVAL&ProductFamily=NVAIEnterprise) through the NVIDIA Developer Program.
+6. An NVIDIA NGC API Key: please follow the guidance in the [NVIDIA NIM Getting Started](https://docs.nvidia.com/nim/large-language-models/latest/getting-started.html#option-2-from-ngc) documentation to generate a properly scoped API key if you haven't already.
+
```
Comment (resker) [inline docs/run.ai/README.md:17]: ## Prerequisites

Required: :heavy_check_mark: 
Provided: :white_check_mark:

|                                                           | Run.ai SaaS                         | Air-gapped   | NVIDIA DGX Cloud |
|-----------------------------------|---------------------|-------------|--------------------|
| A conformant Kubernetes cluster ([RunAI K8s version requirements](https://docs.run.ai/latest/admin/overview-administrator/)) |  :heavy_check_mark: |  :heavy_check_mark: | :white_check_mark: |
| RunAI Control Plane and cluster(s) [installed](https://docs.run.ai/latest/admin/runai-setup/cluster-setup/cluster-install/) and operational |  :heavy_check_mark:  |  :heavy_check_mark: | :white_check_mark: |  
| [NVIDIA GPU Operator](https://github.com/NVIDIA/gpu-operator) installed |  :heavy_check_mark: |  :heavy_check_mark: | :white_check_mark: |
| General NIM requirements: [NIM Prerequisites](https://docs.nvidia.com/nim/large-language-models/latest/getting-started.html#prerequisites |  :heavy_check_mark: |  :heavy_check_mark: |  :heavy_check_mark: |
| An NVIDIA AI Enterprise (NVAIE) License: [Sign up for NVAIE license](https://build.nvidia.com/meta/llama-3-8b-instruct?snippet_tab=Docker&signin=true&integrate_nim=true&self_hosted_api=true) or [Request a Free 90-Day NVAIE License](https://enterpriseproductregistration.nvidia.com/?LicType=EVAL&ProductFamily=NVAIEnterprise) through the NVIDIA  |  :heavy_check_mark: |  :heavy_check_mark: |  :heavy_check_mark: |
| An NVIDIA NGC API Key: please follow the guidance in the [NVIDIA NIM Getting Started](https://docs.nvidia.com/nim/large-language-models/latest/getting-started.html#option-2-from-ngc) documentation to generate a properly scoped API key if you haven't already. |  :heavy_check_mark: |  :heavy_check_mark: |  :heavy_check_mark:|

Required: :heavy_check_mark: 
Provided: :white_check_mark:

**Review by resker — CHANGES_REQUESTED** [review]
Adding knative serving to prerequisites
